src/onegov/org/views/people.py

```diff
@@ -33,13 +33,55 @@ def get_items(self):
 @OrgApp.html(model=Person, template='person.pt', permission=Public)
 def view_person(self, request, layout=None):
 
+    pages = request.session.query(Topic)
+    pages = pages.filter(Topic.people != None).all()
```

Comment:
Note that you should not use `==` and `!=` with `None` in python, with sqlalchemy you can use `is_(None)` and `isnot(None)`: `Topic.people.isnot(None)`